dataprofiler/__init__.py

```diff
@@ -6,6 +6,7 @@
                                     StructuredDataLabeler, \
                                     UnstructuredDataLabeler
 from .validators.base_validators import Validator
+from .dp_logging import set_verbosity
```

Comment:
i would also allow the `get_logger`, e.g. `dp.get_logger` in this functionality.

Reply:
Added